packages/tables/resources/views/columns/text-column.blade.php

```diff
@@ -189,8 +199,25 @@
             @if ($limitedArrayStateCount = count($limitedArrayState ?? []))
                 <{{ $isListWithLineBreaks ? 'li' : 'div' }}
                     class="text-sm text-gray-500 dark:text-gray-400"
+                    x-on:click.prevent="showLimited = ! showLimited"
                 >
-                    {{ trans_choice('filament-tables::table.columns.text.more_list_items', $limitedArrayStateCount) }}
+                    <x-filament::link
+                        color="gray"
+                        tag="button"
+                        x-cloak
+                        x-show="! showLimited"
```

Comment:
@danharrin do we really need `x-cloak` here, as it's shown by default?

Reply:
Hmm sure, you're right